src/ray/common/task/task_common.h

```diff
@@ -0,0 +1,16 @@
+#ifndef RAY_COMMON_TASK_TASK_COMMON_H
+#define RAY_COMMON_TASK_TASK_COMMON_H
+
+#include "ray/protobuf/common.pb.h"
+
+namespace ray {
+
+// Alias `ray::rpc::Language` in `ray` namespace.
+using rpc::Language;
+
+// Alias `ray::rpc::TaskType` in `ray` namespace.
+using rpc::TaskType;
```

Comment:
normally it's not recommended to do this in .h files. It might be fine if there's no other `Language` and `TaskType` definitions in the future.

Reply:
The purpose of doing this is to hide the implementation details that these two enums are defined in protobuf. I'd like other code to use them as if they are defined in `task_common.h`. 
So, in case we need to get rid of protobuf in the future, the refactor would be easier. 

Any idea about how we can better handle this?

Reply:
updated comment